edx_event_bus_redis/internal/producer.py

```diff
@@ -121,7 +128,7 @@ def send(
             stream_data = message.to_binary_dict()
 
             stream = self.client.Stream(full_topic)
-            msg_id = stream.add(stream_data)
+            msg_id = stream.add(stream_data, maxlen=STREAM_MAX_LEN, approximate=True)
```

Comment:
Can you add a comment here about why we force approximate to True, or point to the doc?